### PR TITLE
add alert doc

### PR DESCRIPTION
**Pull request by ldpliu**

### CODE REVIEW

File: doc/README.md [hunk 1]
```diff
@@ -226,15 +226,114 @@ To navigate the global hub dashboards, you can choose to observe and filter the
 
 Similarly, if you want to examine the policy data by `cluster` grouping, begin by using the `Global Hub - Cluster Group Compliancy Overview` dashboard. The navigation flow is identical to the `policy` grouping flow, but you select filters that are related to the cluster, such as managed cluster `labels` and `values`. Instead of viewing policy events for all clusters, after reaching the `Global Hub - What's Changed / Clusters` dashboard, you can view policy events related to an individual cluster.
 
+### Grafana Alerts
+#### Default Grafana Alerts
+We have three alerts by default. These alerts are stored in configmap `multicluster-global-hub-default-grafana-config`. They will watch suspicious policies, suspicious clusters compliance status change and failed cron jobs.
```
Comment (yanmxa) [inline doc/README.md:231]: Is this `configmap` located in namespace `multicluster-global-hub` ?

Comment (ldpliu) [reply doc/README.md:231]: Yes

File: doc/README.md [hunk 2]
```diff
@@ -226,15 +226,114 @@ To navigate the global hub dashboards, you can choose to observe and filter the
...
+1. Suspicious Policy Change
+This Alert rule watches the suspicious policies change, if the following events occur more than 5 times in 1 hour, it becomes a firing alert.
+- A policy was enabled/disabled
```
[the remark this answers is not in the document]
Comment (ldpliu) [reply doc/README.md:234]: > It's easy to reach thousands of policies in an environment. Do you think the rules have performance bottleneck？

We only filter 1 hour data, so it should be ok.

**Comment by yanmxa** [comment]
Looks good to me.

File: doc/README.md [hunk 3]
```diff
@@ -226,15 +226,114 @@ To navigate the global hub dashboards, you can choose to observe and filter the
...
+
+- Data Retention Job Fail
+  If this alert rule becomes firing, it means the [Partition job](#Partition-job) failed. 
+
+#### Delete Default Grafana Alert Rule
+If you want to delete a default grafana alert rule, you need to create the [Customize Grafana Alerting Resources](#Customize-Grafana-Alerting-Resources) and include the `deleteRules`.
+
+#### Customize Grafana Alerts
+##### Customize grafana.ini
+Global hub support customize the grafana.ini files. You just need to create a secret named: `multicluster-global-hub-custom-grafana-config` in the `multicluster-global-hub` namespace, and the data key should be: `grafana.ini`. The following is an example:
```
Comment (yahliu) [inline doc/README.md:257]: suggest to change as "You just need to create a secret, and it must be named `multicluster-global-hub-custom-grafana-config`.

File: doc/README.md [hunk 4]
```diff
@@ -226,15 +226,114 @@ To navigate the global hub dashboards, you can choose to observe and filter the
...
+    skip_verify = true
+    from_address = example@163.com
+    from_name = Grafana	
+    # EHLO identity in SMTP dialog (defaults to instance_name)
+    ;ehlo_identity = dashboard.example.com
+```
+Note: you can not config the section which already in `multicluster-global-hub-default-grafana-config` secret
+
+##### Customize Grafana Alerting Resources 
+Global hub support customize the alerting resources which [grafana support](https://grafana.com/docs/grafana/v9.5/alerting/set-up/provision-alerting-resources/file-provisioning/). You just need to create a configmap named: `multicluster-global-hub-custom-alerting` in `multicluster-global-hub` namespace, and the data key should be: `multicluster-globalhub-alerting.yaml`. The following is an example:
```
Comment (yahliu) [inline doc/README.md:283]: same as above, the confimap name must be called `multicluster-global-hub-custom-alerting`.

Comment (ldpliu) [reply doc/README.md]: done

File: doc/README.md [hunk 5]
```diff
@@ -226,15 +226,114 @@ To navigate the global hub dashboards, you can choose to observe and filter the
...
+- Too many policy events in a cluster
+  For a policy in a cluster, if there are more than 20 events in 5 minutes, it becomes a firing alert. If this alert is always firing, the data in the `event.local_policies` table will increase too fast.
+
+3. Cron Job Failed
+This alert watch the [Cron jobs](#Cronjobs-and-Metrics) failed events. There are two rules in this alert.
+- Local Compliance Job Failed
+  If this alert rule becomes firing, it means the [Local compliance status sync job](#Local-compliance-status-sync-job) failed. It may cause the data to be lost in the `history.local_compliance` table. Please check it and [manually run the job](./how_global_hub_works.md#running-the-summarization-process-manually)
+
+- Data Retention Job Failed
+  If this alert rule becomes firing, it means the [Partition job](#Partition-job) failed.
```
Comment (clyang82) [inline doc/README.md:250]: @yanmxa Do we have a manual step to remediate this issue?

File: doc/README.md [hunk 6]
```diff
@@ -226,15 +226,114 @@ To navigate the global hub dashboards, you can choose to observe and filter the
...
+3. Cron Job Failed
+This alert watch the [Cron jobs](#Cronjobs-and-Metrics) failed events. There are two rules in this alert.
+- Local Compliance Job Failed
+  If this alert rule becomes firing, it means the [Local compliance status sync job](#Local-compliance-status-sync-job) failed. It may cause the data to be lost in the `history.local_compliance` table. Please check it and [manually run the job](./how_global_hub_works.md#running-the-summarization-process-manually)
+
+- Data Retention Job Failed
+  If this alert rule becomes firing, it means the [Partition job](#Partition-job) failed. 
+
+#### Delete Default Grafana Alert Rule
+If you want to delete a default grafana alert rule, you need to create the [Customize Grafana Alerting Resources](#Customize-Grafana-Alerting-Resources) and include the `deleteRules`.
```
Comment (clyang82) [inline doc/README.md:253]: can we show an example to disable all?

File: doc/README.md [hunk 7]
```diff
@@ -226,15 +226,114 @@ To navigate the global hub dashboards, you can choose to observe and filter the
...
+```
+Note: you can not config the section which already in `multicluster-global-hub-default-grafana-config` secret
+
+##### Customize Grafana Alerting Resources 
+Global hub support customize the alerting resources which [grafana support](https://grafana.com/docs/grafana/v9.5/alerting/set-up/provision-alerting-resources/file-provisioning/). You just need to create a configmap in namespace `multicluster-global-hub`, and the configmap name must be: `multicluster-global-hub-custom-alerting`, the configmap data key must be: `multicluster-globalhub-alerting.yaml`. The following is an example:
+
+```yaml
+apiVersion: v1
+data:
+  multicluster-globalhub-alerting.yaml: |
```
Comment (clyang82) [inline doc/README.md:288]: can we just use `alerting` as key?

**Review by clyang82 — CHANGES_REQUESTED** [review]

File: doc/README.md (hunk 5 above)
Comment (yanmxa) [reply doc/README.md:250]: No, We haven't added the manually deletions and creations for this retention job.

**Pull request merged**